client/app/main/about/about.scss

```diff
@@ -1,4 +1,6 @@
 .about-view {
+  overflow-y: auto;
+  @include cfa-scrollbar(5px);
   .developers,
```

Comment:
@include is for using a mixin.

http://krasimirtsonev.com/blog/article/SASS-mixins-extends-and-placeholders-differences-use-cases
http://roytomeij.com/blog/2013/should-you-use-a-sass-mixin-or-extend.html
https://tech.bellycard.com/blog/sass-mixins-vs-extends-the-data/